_includes/hero.html

```diff
@@ -1,59 +1,57 @@
 {% macro render(hero) %}
 
 {% from "post-macros.html" import category_icon as category_icon %}
-    
+
 {% if hero['related_post'] %}
     {% set related_post = get_document('posts', hero['related_post']) %}
 {% endif %}
 
+{% if related_post %}
+    {% set hero_href = ['href=', related_post.permalink] %}
+{% elif hero.custom_fields.link %}
+    {% set hero_href = ['href=', hero.custom_fields.link[0]] %}
+{% endif %}
+
 <section class="content_hero hero"
          style="background-color: {{ hero.custom_fields.background_color[0]
                                      if hero.custom_fields.background_color }}">
     <div class="hero_wrapper">
-    {% if related_post %}
-        <a class="hero_card" href="{{ related_post.permalink }}">
-    {% elif hero.custom_fields.link %}
-        <a class="hero_card" href="{{ hero.custom_fields.link[0] }}">
-    {% else %}
-        <a class="hero_card">
-    {% endif %}
-            <div class="hero_card-bg">
-                <div class="hero_card-padding">
+        <a class="hero_card" {{ hero_href|join if hero_href }}>
```

Comment:
- Yeah, took a little while to figure out how to combine strings and vars in Jinja
- Yep, automagically. I included the quotes when setting the var but it printed them twice. Don't ask me how it works but it works ;)